tests/codemod/deprecate-version/test-data.ts

```diff
@@ -0,0 +1,35 @@
+export const inputCode = `
```

Comment:
the test data can include more scenarios to test the codemod's full functionality, for example
- aliased imports e.g.
```
import { MediaWidths as DSMediaWidth } from "@lifesg/react-design-system"
```
- multiple imports e.g.
```
import { Text, TextStyleHelper } from ...
```
- duplicate modules e.g.
```
import { Text } from "@lifesg/react-design-system/text";
import { TextStyleHelper } from "@lifesg/react-design-system/text";
```
- from the top level
```
import { MediaWidths } from "@lifesg/react-design-system";
```